pkg/storage/minio/deleter.go

```diff
@@ -23,17 +23,17 @@ func (v *storageImpl) Delete(ctx context.Context, module, version string) error
 
 	versionedPath := v.versionLocation(module, version)
 
-	modPath := fmt.Sprintf("%s/go.mod", versionedPath)
+	modPath := fmt.Sprintf("%s.mod", versionedPath)
```

Comment:
same here and below regarding `versionedPath`?

Reply:
Yes! That is why I mentioned it is a breaking change :(
The reason being: https://github.com/gomods/athens/pull/1035/files#diff-617c9c1a2f86faafdb72e42ccd20a502R46 where `config.PackageVersionPath` has its own function of storing it and minio will use the moduploader which uses this function.